<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="TX590-5R10QN2" name="Default test" tests="1" failures="1" timestamp="14 May 2018 20:43:25 GMT" time="3.652" errors="0">
  <testcase name="runTC_SF_APACCreateNewMarketingList" time="3.621" classname="testscripts.SF.SprintR48.TC_SF_ABAPACBrokerCreateNewMarketingList">
    <failure type="java.lang.AssertionError" message="java.lang.NullPointerException">
      <![CDATA[java.lang.AssertionError: java.lang.NullPointerException
	at org.testng.Assert.fail(Assert.java:94)
	at com.cognizant.Craft.CRAFTTestCase.tearDownTestRunner(CRAFTTestCase.java:149)
	at testscripts.SF.SprintR48.TC_SF_ABAPACBrokerCreateNewMarketingList.runDriverScript(TC_SF_ABAPACBrokerCreateNewMarketingList.java:34)
	at testscripts.SF.SprintR48.TC_SF_ABAPACBrokerCreateNewMarketingList.runTC_SF_APACCreateNewMarketingList(TC_SF_ABAPACBrokerCreateNewMarketingList.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- runTC_SF_APACCreateNewMarketingList -->
</testsuite> <!-- Default test -->
